cram_common/cram_manipulation_interfaces/src/manipulation-interfaces.lisp

```diff
@@ -81,3 +81,11 @@ is considered closed.
 `container-designator' is a designator describing the container.")
   (:method :heuristics 20 (container-name)
     nil))
+
+(defgeneric get-z-offset-for-placing-distance (other-object object attachment)
```

Comment:
This should not be a general knowledge query. It's only used in our heuristics calculation for the placing trajectory. So it's rather heuristics-engine specific. So move this to where the get-object-type-grasp-transforms is defined, i.e. trajectories.lisp i guess.

Reply:
Rename and exchange parameter order.